mantidproject/mantid
# SWANS loader

This is a temporary loader. No release notes.

To test you can run the Test:

```
make -j22 DataHandlingTest &&  ctest -VV --output-on-failure -R DataHandlingTest_LoadSwans
```

Or in Mantid select the algorithm: `LoadSwans` and load the file whose md5 is `Testing/Data/UnitTest/SWANS_RUN80814.dat.md5`.
You can see the instrument view. It shows a rotation of the detector.

At the SNS, the following python code can be run.

```
LoadSwans(FilenameData='/SNS/VULCAN/IPTS-16013/shared/SANS_detector/RUN80834.dat', FilenameMetaData='/SNS/VULCAN/IPTS-16013/shared/SANS_detector/RUN80834_meta.dat', OutputWorkspace='t2')

LoadSwans(FilenameData='/SNS/VULCAN/IPTS-16013/shared/SANS_detector/RUN80814.dat', FilenameMetaData='/SNS/VULCAN/IPTS-16013/shared/SANS_detector/RUN80814_meta.dat', OutputWorkspace='t1')
```

Observe the detector rotation for `t1` in the instrument view.
